### ISSUE TITLE
Error message on a Git graph

### ISSUE THREAD

**Issue by piRGoif** [mjbvz/vscode-markdown-mermaid#227]
Hello,

Trying to edit our version history using latest vs code and your extension, but I get an error message in the preview

File : https://github.com/Combodo/iTop/blob/develop/.doc/itop-version-history.md

With the existing file state (commit d8da119) I get in VSCode preview : 
> Trying to checkout branch which is not yet created. (Help try using "branch develop")

If I add `branch develop` just below `gitGraph` line 6 I get : 
> Trying to create an existing branch. (Help: Either use a new name if you want create a new branch or try using "checkout develop")

If I replace `branch develop` by `checkout develop` I get : 
> Trying to checkout branch which is not yet created. (Help try using "branch develop")

**Comment by piRGoif** [comment]
Hello,
Yes still the same error on the HEAD develop version (commit 4fb581c)

In my previous message I was referring to commit d8da119, this is : 
https://github.com/Combodo/iTop/blob/d8da119c56df256b5e2f7287a60eecb6a37cd10e/.doc/itop-version-history.md

Current file state, with the same error still : 
https://github.com/Combodo/iTop/blob/4fb581c784061ae348ea6d19e70248ec1bfeb310/.doc/itop-version-history.md

**Comment by mjbvz** [comment]
This is what I see on the current release:

![Image](https://github.com/mjbvz/vscode-markdown-mermaid/assets/12821956/cd4eb6c8-25ee-4e4e-8b9d-278e0446eaca)

**Comment by piRGoif** [comment]
Ok, thanks for the test !
So there is something different on our configurations either on version or parameters ?

On my side : 
- VSCode 1.84.2
- Markdown Preview Mermaid Support 1.21.0
- doesn't seems there are much changes in the extension configuration ?

**Comment by mjbvz** [comment]
Hm, very strange. Not sure what is going on but two follow up questions:

- Do you have any other extensions installed? Can you try disabling them to see if they cause the issue
- What operating system are you on. I'm wondering if this could be related to line endings

**Comment by piRGoif** [comment]
Yes, lots of extensions installed... I'll have to try, will let you know

But I can answer immediately about line endings : running on a french Windows 10, but git clones are done using `autocrlf = input`. And indeed the history file is recognized as Lf both by VSCode and Notepad++

**Comment by piRGoif** [comment]
As I was about to test by disabling other extensions... I'm just seeing the file now correctly rendered !
But... VSCode was updated in the afternoon, now running 1.85.0.
Maybe there was an incompatibility with my previous version 1.84.2 ? 
Anyway for now everything is ok, thanks !
